interfaces/cython/cantera/examples/reactors/pfr.py

```diff
@@ -0,0 +1,174 @@
+# -*- coding: utf-8 -*-
+"""
+This example solves a plug-flow reactor problem of hydrogen-oxygen combustion.
+The PFR is computed by two approaches: The simulation of a Lagrangeian fluid
+particle, and the simulation of a chain of reactors.
+"""
```

Comment:
You're right, the Lagrangian approach is much faster. Additionally, it is much less sensitive to the number of steps used. This makes sense, because the Lagrangian approach is for all `advance` steps discretized by the tolerances of the internal time stepping, whereas for the reactor chain the discretization does have an effect.

| Steps | Lagrangian | Reactor Chain |
|-------|------------|---------------|
| 20    | .0739 s    | .1961 s       |
| 200   | .0776 s    | .6291 s       | 
| 2000  | .1404 s    |   3.149 s           |

![pfr_t_z](https://cloud.githubusercontent.com/assets/6306137/8495499/5a72f134-216e-11e5-9a5d-24ffe039ae16.png)
![pfr_xh2_t](https://cloud.githubusercontent.com/assets/6306137/8495500/5a73e486-216e-11e5-9403-5b6f5eb4ceef.png)